### ISSUE TITLE
Offline Mode

### ISSUE THREAD

**Issue by pranavAbe** [Pong420/google-tasks-desktop#52]
A very useful feature that could set this desktop client apart from any other implementation of Google Tasks Desktop could be the implementation of an offline mode. Being able to create, edit, and delete tasks offline and have them sync when a network connection is available is a very useful feature that could be considered for the roadmap of this project.

**Comment by Pong420** [comment]
Hi, which version of are you using?

**Comment by alancphilpott** [comment]
> A very useful feature that could set this desktop client apart from any other implementation of Google Tasks Desktop could be the implementation of an offline mode. Being able to create, edit, and delete tasks offline and have them sync when a network connection is available is a very useful feature that could be considered for the roadmap of this project.

I agree with this. If I find the time down the line and the dev is open to reviewing pull requests, I'd be more than happy to work on this feature.
